From 91f327c30a9f4c50bd739d2c8cc6a8377cc3be51 Mon Sep 17 00:00:00 2001
From: Caleb Connolly <caleb@connolly.tech>
Date: Mon, 30 May 2022 14:41:56 +0100
Subject: [PATCH 053/142] arm64: dts: qcom: sdm845-shift-axolotl: set TS i2c
 bus clock

Avoids the message "geni_i2c 894000.i2c: Bus frequency not specified, default to 100kHz."
and sets a higher rate than the default for good responsiveness.

Signed-off-by: Caleb Connolly <caleb@connolly.tech>
---
 arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts b/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
index 949685658b..092b63bb5e 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
+++ b/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
@@ -434,6 +434,7 @@ zap-shader {
 
 &i2c5 {
 	status = "okay";
+	clock-frequency = <400000>;
 
 	touchscreen@38 {
 		compatible = "focaltech,fts8719";
-- 
2.49.0

